src/test/java/com/artipie/test/UrlCredsHelper.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.test;
+
+import com.artipie.ArtipieServer;
+
+/**
+ * Class to simplify working with url that may contains user credentials.
+ * @since 0.12
+ */
+public final class UrlCredsHelper {
```

Comment:
@genryxy that's bad class name, please think of something better. Maybe it is "repository url"? Or maybe this is not an entity and it will make more sense to add `url()` method to `ArtipieServer` or `TestContainer`

Reply:
@olegmoz thanks. I'm not sure that adding `url()` to `ArtipieServer` or `TestContainer` will be better because in npm it uses for creating config file before creating server and testcontainer. There is only one such case but there is one. `UrlCntnCreds` is more suitable?

Reply:
@genryxy I think `RepositoryUrl` would be most suitable name. I would also rename `url()` method to `string()` in this case